aws-kendra-index/src/main/java/software/amazon/kendra/index/ListHandler.java

```diff
@@ -38,4 +46,16 @@ public ProgressEvent<ResourceModel, CallbackContext> handleRequest(
                 .status(OperationStatus.SUCCESS)
                 .build();
     }
+
+  private ListIndicesResponse listIndices (ListIndicesRequest listIndicesRequest, ProxyClient<KendraClient> proxyClient) {
+    try {
+      return proxyClient.injectCredentialsAndInvokeV2(listIndicesRequest, proxyClient.client()::listIndices);
+    } catch (AccessDeniedException e) {
+      throw new CfnAccessDeniedException(LIST_INDICES, e);
+    } catch (ThrottlingException e) {
+      throw new CfnThrottlingException(e);
```

Comment:
How come we don't pass LIST_INDICES when creating CfnThrottlingException here?

Reply:
Yeah that's inconsistent, will update.